api/api/utils/image_proxy/__init__.py

```diff
@@ -75,11 +80,40 @@ def get_request_params_for_extension(
     )
 
 
-def get(
+@sync_to_async
+def _tally_response(
+    tallies,
+    media_info: MediaInfo,
+    month: str,
+    domain: str,
+    response: aiohttp.ClientResponse,
+):
+    """
+    Tally image proxy response without waiting for Redis to respond.
+
+    Pulled into a separate function to help reduce overload when skimming
+    the `get` function, which is complex enough as is.
+    """
+    tallies.incr(f"thumbnail_response_code:{month}:{response.status}"),
+    tallies.incr(
+        f"thumbnail_response_code_by_domain:{domain}:" f"{month}:{response.status}"
+    )
+    tallies.incr(
+        f"thumbnail_response_code_by_provider:{media_info.media_provider}:"
+        f"{month}:{response.status}"
+    )
+
+
+_UPSTREAM_TIMEOUT = aiohttp.ClientTimeout(15)
+
+
+async def get(
     media_info: MediaInfo,
     request_config: RequestConfig = RequestConfig(),
 ) -> HttpResponse:
     """
+    Retrieve proxied image from site accelerator.
+
```

Comment:
It seems that we should prioritize the issue of renaming all instances of Photon to `site accelerator` to keep consistency: a single function's docstrings use both names now.

Reply:
Good point. This is such a significant change already, I'll just switch this back to use Photon or maybe just avoid mention of the service used. We already have an issue to cover this (#2542), and it's better not to mix the names up until we're actively fixing it everywhere.

Reply:
Fixed in https://github.com/WordPress/openverse/pull/3388/commits/1d6ebd52f5759a93a7c7cae346faef8cc3726f02